purge: explicitly provide fsid for cephadm shell

dsavineau opened the pull request:
The fsid value is already mandatory for running the purge playbook and
we provide the value to the rm-cluster command.
Let's do the same for the cephadm shell command.

Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1980012

Signed-off-by: Dimitri Savineau <dsavinea@redhat.com>

guits commented:
@dsavineau why not patching this on the `devel` branch?

dsavineau commented:
> @dsavineau why not patching this on the `devel` branch?

Because that task doesn't exist

guits commented:
this one ? https://github.com/ceph/cephadm-ansible/blob/devel/cephadm-purge-cluster.yml#L48-L49

dsavineau commented:
> this one ? https://github.com/ceph/cephadm-ansible/blob/devel/cephadm-purge-cluster.yml#L48-L49

Indeed.

I've probably forgot to rebase my local devel branch 😔

dsavineau committed:
purge: explicitly provide fsid for cephadm shell

The fsid value is already mandatory for running the purge playbook and
we provide the value to the rm-cluster command.
Let's do the same for the cephadm shell command.

Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1980012

Signed-off-by: Dimitri Savineau <dsavinea@redhat.com>
